7/9 provisioning tests fail/error on manual run instructions

- [x] Tests are passing on e2e

But while manually running provisioning tests on Mac/unix, 7/9 tests fail on various errors likely due to the missing .config volumes that persist credentials

Steps to reproduce: 
1. Provision sandbox on personal account
2. docker run [per instructions](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/tests/provisioning)

[Full error log output](https://docs.google.com/document/d/1pA6bS30_S8VWhfS5_OyUxJnnX6RhE3Zt9P_U5eiH9ho/edit?hl=en)
**2/9 tests passing:** 
```
to select an already authenticated account to use.
testNodeMachineType (__main__.TestGKECluster)
Test if the machine type for the nodes is as specified ... ERROR
testNumberOfNode (__main__.TestGKECluster)
Test if the number of nodes in the node pool is as specified ... ERROR
testReachOfHipsterShop (__main__.TestGKECluster)
Test if querying hipster shop returns 200 ... ERROR
testStatusOfServices (__main__.TestGKECluster)
Test if all the service deployments are ready ... ERROR
testDifferentZone (__main__.TestLoadGenerator)
Test if load generator is in a different zone from the GKE cluster ... ok
testNumberOfLoadgen (__main__.TestLoadGenerator)
Test if there's only one load generator instance ... ok
testReachOfLoadgen (__main__.TestLoadGenerator)
Test if querying load generator returns 200 ... ERROR
testAPIEnabled (__main__.TestProjectResources)
Test if all APIs requested are enabled ... FAIL
testErrorReporting (__main__.TestProjectResources)
Test if we can report error using Error Reporting API ... ERROR
```

Note: I have an active, credentialed GCloud user account. Though some errors could still be set up issues on my end.

Update: per suggestions from Levi to run on Cloud Shell instead, only this test was failling: 

```bash
Test if load generator is in a different zone from the GKE cluster ... FAIL

======================================================================
FAIL: testDifferentZone (__main__.TestLoadGenerator)
Test if load generator is in a different zone from the GKE cluster
----------------------------------------------------------------------
Traceback (most recent call last):
  File "provision_test.py", line 86, in testDifferentZone
    self.assertTrue(zone != getClusterZone())
AssertionError: False is not true
----------------------------------------------------------------------

```

--- a/tests/provisioning/README.md
+++ b/tests/provisioning/README.md
@@ -1,6 +1,6 @@
 # Tests for Cloud Ops Sandbox Provisioning
 
-This test is integrated into the Github CI system. If you want to run it manually, please follow the steps below and *make sure you are running on the Cloud Shell*.
+This test is integrated into the Github CI system. If you want to run it manually, please follow the steps below and **make sure you are running on the Cloud Shell**.
 
 1. Build docker
 ```bash

--- a/tests/provisioning/README.md
+++ b/tests/provisioning/README.md
@@ -1,6 +1,6 @@
 # Tests for Cloud Ops Sandbox Provisioning
 
-This test is integrated into the Github CI system. If you want to run it manually, please follow the steps below.
+This test is integrated into the Github CI system. If you want to run it manually, please follow the steps below and *make sure you are running on the Cloud Shell*.
 
 1. Build docker
 ```bash
@@ -9,5 +9,5 @@ docker build -t $image_name .
 
 2. Run docker
 ```bash
-docker run -rm -e GOOGLE_CLOUD_PROJECT=$project_id -e ZONE=$gke_zone $image_name
+docker run --rm -e GOOGLE_CLOUD_PROJECT=$project_id -e ZONE=$gke_zone $image_name
 ```

